charts/codesealer/README.md

```diff
@@ -158,6 +159,8 @@ helm install codesealer ${CODESEALER_HELM_CHART} \
   --set worker.ingress.nodePort="${INGRESS_NODEPORT}" \
   --set worker.redis.namespace="${REDIS_NAMESPACE}" \
   --set worker.config.bootloader.redisPassword="${REDIS_PASSWORD}" \
+  --set sidecar.enabled=true \
+  --set environment=${CODESEALER_ENV} \
```

Comment:
What is this? I don't see a mention of `CODESEALER_ENV` anywhere else?

Reply:
Not really sure, suppose it came in through a rebase. Removed